__tests__/app.test.js

```diff
@@ -113,18 +113,19 @@ describe("GET /api/articles", () => {
 describe("GET /api/articles/:article_id", () => {
     test("article object contains all correct properties and has specified ID", () => {
         return request(app)
-            .get("/api/articles/7")
+            .get("/api/articles/9")
```

Comment:
we really want to add a new test for this, if we **had** to change the test from before it was too brittle